### ISSUE TITLE
FYI, the MMRM module from the [teal.gallery](https://genentech.shinyapps.io/nest_efficacy_stable/) is showing some errors as well. I tried out with different user input values.

### ISSUE THREAD

**Issue by pawelru** [insightsengineering/teal.gallery#108]
FYI, the MMRM module from the [teal.gallery](https://genentech.shinyapps.io/nest_efficacy_stable/) is showing some errors as well. I tried out with different user input values. 

<img width="904" alt="image" src="https://github.com/insightsengineering/tlg-catalog/assets/87760125/e9366104-567e-4eb5-97dd-cac8096e19d4">

_Originally posted by @imazubi in https://github.com/insightsengineering/tlg-catalog/issues/122#issuecomment-1821157983_

**Comment by shajoezhu** [comment]
this can be resolved with updating the renv file 

see example at https://github.com/shajoezhu/china_nest_teal_gee_mmrm_demo/blob/main/renv.lock#L2714

**Comment by pawelru** [comment]
I don't think developers are expected to change `.lock` files. Those are auto-generated and auto-updated so it might happen that your changes will be immidiately overwritten by auto-update. Am I right @vedhav?

**Comment by vedhav** [comment]
Yes, The `renv.lock` files shown in the `teal.gallery` repo are not deployed in the teal.gallery demo deployments.

- The `_stable` deployments always install the `teal*` (and any GitHub packages in the renv.lock) packages from the latest tag version
- The `_dev` deployments always install the `teal*` packages from the `main` branch (while the teal.gallery app code is taken from the `dev` branch)

Once we deploy the apps using the CI, the last step is to commit the updated renv.lock files which is failing right now because of no permission to commit on main which is causing this false perception about the deployments. This will soon be fixed in this [PR](https://github.com/insightsengineering/teal.gallery/pull/106/files#diff-28802fbf11c83a2eee09623fb192785e7ca92a3f40602a517c011b947a1822d3R25-R26)
